.github/workflows/build_deploy.yml

```diff
@@ -0,0 +1,74 @@
+name: build_deploy
+
+on:
+  push:
+    branches: master
+  pull_request:
+  repository_dispatch:
+  workflow_dispatch:
+
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+# Allow only one concurrent deployment, skipping runs queued between the run in-progress and latest queued.
+# However, do NOT cancel in-progress runs as we want to allow these production deployments to complete.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: false
+
+jobs:
+  build:
+    name: Build site
+    runs-on: ubuntu-latest
```

Comment:
I think we can just directly use the Metanorma container to compile documents since it contains all the necessary gems.